{"nbformat":4,"nbformat_minor":0,"metadata":{"colab":{"name":"Crawler_Pages.ipynb","provenance":[],"authorship_tag":"ABX9TyMVJ1kPWrYyliouCjVe4/at"},"kernelspec":{"name":"python3","display_name":"Python 3"},"language_info":{"name":"python"}},"cells":[{"cell_type":"code","metadata":{"id":"MF3pSFCEqaK3","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1620341414990,"user_tz":300,"elapsed":21429,"user":{"displayName":"Swathi Bodavula","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjEnLhW6QQwYx5lzdBRurwQeXMmO-cdAZHAmMnI=s64","userId":"10131213081504042480"}},"outputId":"671189e3-92a4-427e-acc7-21b125b61787"},"source":["from google.colab import drive\n","drive.mount('/content/gdrive')"],"execution_count":null,"outputs":[{"output_type":"stream","text":["Mounted at /content/gdrive\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"c3N5tL_z4FBE"},"source":["from google.colab import drive\n","drive.mount('/content/drive')"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"y9cIWLTlkb29"},"source":["import os\n","import pickle\n","import requests\n","from collections import deque\n","from bs4 import BeautifulSoup"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"cBr0L-DwqhdA"},"source":["print(os.getcwd())\n","path = \"/content/gdrive/MyDrive/CS-582_project\"\n","os.chdir(path)"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"gsW7K-zDrUoH"},"source":["# number of pages to crawl\n","page_crawl_limit = 6000"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"pUZiMajxvNv7"},"source":["# to make sure error log file is initially empty\n","error_file = \"error_log.txt\"\n","f = open(error_file, \"w+\")\n","f.close()"],"execution_count":null,"outputs":[]},{"cell_type":"markdown","metadata":{"id":"sSokIDPxpcet"},"source":["**Domain for the crawler**"]},{"cell_type":"code","metadata":{"id":"cHUhYz14pKGv"},"source":["domain = \"uic.edu\"\n","\n","# starting url\n","init_url = \"https://cs.uic.edu\"\n","\n","# folder to store retrieved web pages\n","pages_directory_path = \"./RetrievedPages/\"\n","\n","ignored_exten = [\n","    \".xlsx\",\n","    \".css\",\n","    \".js\",\n","    \".aspx\",\n","    \".png\",\n","    \".jpg\",\n","    \".jpeg\",\n","    \".gif\",\n","    \".svg\",\n","    \".ico\",\n","    \".mp4\",\n","    \".avi\",\n","    \".tar\",\n","    \".gz\",\n","    \".tgz\",\n","    \".zip\",\n","    \".pdf\",\n","    \".doc\",\n","    \".docx\",\n","    \".ppt\",\n","    \".pptx\",\n","    \".xls\"]"],"execution_count":null,"outputs":[]},{"cell_type":"code","metadata":{"id":"FN9M5xrDu93x","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1619875565443,"user_tz":300,"elapsed":665,"user":{"displayName":"Swathi Bodavula","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjEnLhW6QQwYx5lzdBRurwQeXMmO-cdAZHAmMnI=s64","userId":"10131213081504042480"}},"outputId":"3a6e4c4e-1118-4c31-840a-fc5de6c9ba92"},"source":["#  Breadth First Search web traversal for URL'S\n","url_queue = deque()\n","url_queue.append(init_url)\n","\n","# storing traversed URLs\n","crawled_urls = []\n","crawled_urls.append(init_url)\n","\n","# dict to track pages fetched and stored in folder\n","crawled_pages = {}\n","page_num = 1\n","\n","url_queue"],"execution_count":null,"outputs":[{"output_type":"execute_result","data":{"text/plain":["deque(['https://cs.uic.edu'])"]},"metadata":{"tags":[]},"execution_count":10}]},{"cell_type":"code","metadata":{"id":"wv9EDf0-xM7m","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1619890224287,"user_tz":300,"elapsed":14659505,"user":{"displayName":"Swathi Bodavula","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjEnLhW6QQwYx5lzdBRurwQeXMmO-cdAZHAmMnI=s64","userId":"10131213081504042480"}},"outputId":"014c4bf2-0ceb-4860-f36c-736ea1ba0ac5"},"source":["while url_queue:\n","  try:\n","    # fetch the first url from the queue\n","    url = url_queue.popleft()\n","    # HTML code\n","    request = requests.get(url)\n","    # print(request)\n","\n","    if request.status_code == 200:\n","\n","      parser = BeautifulSoup(request.text, \"lxml\")\n","      # anchor tags\n","      extracted_tags = parser.find_all(\"a\")\n","      # print(extracted_tags)\n","\n","      # print(\"*********************\")\n","\n","      if len(extracted_tags) != 0:\n","        crawled_pages[page_num] = url\n","        file_name = pages_directory_path + str(page_num)\n","\n","        # create a directory if RetrievedPages directory doesn't exist,\n","        # if exists no changes made to the directory\n","        os.makedirs(os.path.dirname(file_name), exist_ok=True)\n","\n","        # save the retrievd web page from the current url\n","        with open(file_name, \"w\", encoding=\"utf-8\") as f:\n","            f.write(request.text)\n","        f.close()\n","\n","        for tags in extracted_tags:\n","          ref_url = tags.get(\"href\")\n","  \n","\n","          if (ref_url is not None and ref_url.startswith(\"http\")):\n","            if (not any (i in ref_url for i in ignored_exten)):\n","              ref_url = ref_url.lower()\n","              ref_url = ref_url.split('#')[0]\n","              ref_url = ref_url.split('?', maxsplit=1)[0]\n","              ref_url = ref_url.rstrip('/')\n","              ref_url = ref_url.strip()\n","\n","            if ref_url not in crawled_urls and domain in ref_url:\n","              url_queue.append(ref_url)                 \n","              crawled_urls.append(ref_url)\n","            \n","        if (len(crawled_pages) > page_crawl_limit):\n","          break     \n","        page_num += 1        \n","\n","  except Exception as e:\n","      with open(error_file, \"a+\") as f:\n","                  # add error message to error log\n","        f.write(f\"Unable to connect to {url}\")\n","        f.write(f\"\\nError occured: {e}\\n\\n\")\n","      f.close()\n","\n","      print(\"Unable to connect to \", url)\n","      print(\"Error occured: \", e, \" \\n\")\n","      continue\n","\n","\n","\n"],"execution_count":null,"outputs":[{"output_type":"stream","text":["Unable to connect to  https://give.engineering.uic.edu\n","Error occured:  HTTPSConnectionPool(host='give.engineering.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://emergency.uic.edu\n","Error occured:  HTTPSConnectionPool(host='emergency.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://outlook.uic.edu\n","Error occured:  HTTPSConnectionPool(host='outlook.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://emergency.uic.edu\n","Error occured:  HTTPSConnectionPool(host='emergency.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://give.uic.edu\n","Error occured:  HTTPSConnectionPool(host='give.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://gat.uic.edu\n","Error occured:  HTTPSConnectionPool(host='gat.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://tag.uic.edu\n","Error occured:  HTTPSConnectionPool(host='tag.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/social-media-directory\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /social-media-directory (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://fmweb.fm.uic.edu/Trans/paratransit.aspx\n","Error occured:  HTTPConnectionPool(host='fmweb.fm.uic.edu', port=80): Max retries exceeded with url: /Trans/paratransit.aspx (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5aa3988a90>: Failed to establish a new connection: [Errno -2] Name or service not known'))  \n","\n","Unable to connect to  https://pharmacy.uic.edu\n","Error occured:  HTTPSConnectionPool(host='pharmacy.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://vcsaonline.uic.edu/lms\n","Error occured:  HTTPSConnectionPool(host='vcsaonline.uic.edu', port=443): Max retries exceeded with url: /lms (Caused by SSLError(SSLCertVerificationError(\"hostname 'vcsaonline.uic.edu' doesn't match 'dev.sa.uic.edu'\")))  \n","\n","Unable to connect to  https://events.uic.edu\n","Error occured:  HTTPSConnectionPool(host='events.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://password.uic.edu\n","Error occured:  HTTPSConnectionPool(host='password.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/uicnews/section/circleinterchange\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /uicnews/section/circleinterchange (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/communicating-on-campus/photo-video-policies\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /communicating-on-campus/photo-video-policies (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://https://today.uic.edu/coronavirus\n","Error occured:  HTTPConnectionPool(host='https', port=80): Max retries exceeded with url: //today.uic.edu/coronavirus (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5aa45d6c50>: Failed to establish a new connection: [Errno -2] Name or service not known'))  \n","\n","Unable to connect to  https://web-travel-prod.cc.uic.edu/index.html\n","Error occured:  HTTPSConnectionPool(host='web-travel-prod.cc.uic.edu', port=443): Max retries exceeded with url: /index.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/state-authorization\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /state-authorization (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://docs.financialaid.uic.edu/index.html\n","Error occured:  HTTPSConnectionPool(host='docs.financialaid.uic.edu', port=443): Max retries exceeded with url: /index.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/uicnews\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /uicnews (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://makerspace.uic.edu\n","Error occured:  HTTPSConnectionPool(host='makerspace.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://acer.uic.edu\n","Error occured:  HTTPSConnectionPool(host='acer.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://help.uic.edu\n","Error occured:  HTTPSConnectionPool(host='help.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://embs.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='embs.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://logica.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='logica.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://advis-lab.red.uic.edu\n","Error occured:  HTTPSConnectionPool(host='advis-lab.red.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://wics.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='wics.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://theory.cs.uic.edu\n","Error occured:  HTTPSConnectionPool(host='theory.cs.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://hci.cs.uic.edu\n","Error occured:  HTTPSConnectionPool(host='hci.cs.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://cs.red.uic.edu/profiles/bing-liu\n","Error occured:  HTTPSConnectionPool(host='cs.red.uic.edu', port=443): Max retries exceeded with url: /profiles/bing-liu (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://sisl.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='sisl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://cs.red.uic.edu\n","Error occured:  HTTPSConnectionPool(host='cs.red.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://aes.uic.edu/aes-staff\n","Error occured:  HTTPSConnectionPool(host='aes.uic.edu', port=443): Max retries exceeded with url: /aes-staff (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://webmail.uic.edu\n","Error occured:  HTTPSConnectionPool(host='webmail.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/assistive-technology-certificate-program\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /assistive-technology-certificate-program (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://www.psych.uic.edu\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.psych.uic.edu/research/center-on-mental-health-services-research-and-policy/about-us/about-the-center\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: /research/center-on-mental-health-services-research-and-policy/about-us/about-the-center (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.psych.uic.edu/research/center-on-mental-health-services-research-and-policy/our-team\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: /research/center-on-mental-health-services-research-and-policy/our-team (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://archive.news.uic.edu\n","Error occured:  HTTPSConnectionPool(host='archive.news.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://red.uic.edu/resources/uic-web-strategy-standards\n","Error occured:  HTTPSConnectionPool(host='red.uic.edu', port=443): Max retries exceeded with url: /resources/uic-web-strategy-standards (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://red.uic.edu/resources/uic-web-content-style-guide\n","Error occured:  HTTPSConnectionPool(host='red.uic.edu', port=443): Max retries exceeded with url: /resources/uic-web-content-style-guide (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://emergency.uic.edu/uic-alerts\n","Error occured:  HTTPSConnectionPool(host='emergency.uic.edu', port=443): Max retries exceeded with url: /uic-alerts (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://logos.uic.edu/index.cgi\n","Error occured:  HTTPSConnectionPool(host='logos.uic.edu', port=443): Max retries exceeded with url: /index.cgi (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://security.uic.edu/policies\n","Error occured:  HTTPSConnectionPool(host='security.uic.edu', port=443): Max retries exceeded with url: /policies (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.hr.uic.edu/policies\n","Error occured:  HTTPSConnectionPool(host='www.hr.uic.edu', port=443): Max retries exceeded with url: /policies (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/communicating-on-campus\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /communicating-on-campus (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/projectmanagement\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /projectmanagement (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://ec.uic.edu/brian-porter\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /brian-porter (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/communicating-on-campus/contact-us\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /communicating-on-campus/contact-us (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://jmls-dev.red.uic.edu/experiential-learning\n","Error occured:  HTTPSConnectionPool(host='jmls-dev.red.uic.edu', port=443): Max retries exceeded with url: /experiential-learning (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ccslgbtqpa.uic.edu/files/2016/06/Attendance-At-Approved-Events-Policy.pdf\n","Error occured:  HTTPSConnectionPool(host='ccslgbtqpa.uic.edu', port=443): Max retries exceeded with url: /files/2016/06/Attendance-At-Approved-Events-Policy.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://go.uic.edu/pickup_upass\n","Error occured:  HTTPSConnectionPool(host='cas.uic.edu', port=443): Max retries exceeded with url: /upass_pickup/index2.php (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://coeinmch.uic.edu/masks-for-moms\n","Error occured:  HTTPSConnectionPool(host='coeinmch.uic.edu', port=443): Max retries exceeded with url: /masks-for-moms (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://pharmacy.uic.edu/covid19\n","Error occured:  HTTPSConnectionPool(host='pharmacy.uic.edu', port=443): Max retries exceeded with url: /covid19 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://uictechcenter.uic.edu\n","Error occured:  HTTPSConnectionPool(host='techcenter.uic.edu%20', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa47012d0>: Failed to establish a new connection: [Errno -2] Name or service not known'))  \n","\n","Unable to connect to  https://cas.uic.edu/hhall\n","Error occured:  HTTPSConnectionPool(host='cas.uic.edu', port=443): Max retries exceeded with url: /hhall (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://tigger.uic.edu/depts/oar/undergrad/transfer_requirements_undergrad.html\n","Error occured:  ('Connection aborted.', OSError(0, 'Error'))  \n","\n","Unable to connect to  http://dos.red.uic.edu/student-veterans-affairs/contact-student-veteran-affairs\n","Error occured:  HTTPSConnectionPool(host='dos.red.uic.edu', port=443): Max retries exceeded with url: /student-veterans-affairs/contact-student-veteran-affairs (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://cas.uic.edu/hr/careersinstudentaffairs.shtml\n","Error occured:  HTTPSConnectionPool(host='cas.uic.edu', port=443): Max retries exceeded with url: /hr/careersinstudentaffairs.shtml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://commuter.uic.edu\n","Error occured:  HTTPSConnectionPool(host='commuter.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://vpua.uic.edu/nikos-varelas-vice-provost-for-undergraduate-affairs\n","Error occured:  HTTPSConnectionPool(host='vpua.uic.edu', port=443): Max retries exceeded with url: /nikos-varelas-vice-provost-for-undergraduate-affairs (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://ec.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.hr.uic.edu\n","Error occured:  HTTPSConnectionPool(host='www.hr.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://exchange.uic.edu/owa/auth/logon.aspx?replaceCurrent=1&url=https%3a%2f%2fexchange.uic.edu%2fowa%2f\n","Error occured:  HTTPSConnectionPool(host='exchange.uic.edu', port=443): Max retries exceeded with url: /owa/auth/logon.aspx?replaceCurrent=1&url=https%3a%2f%2fexchange.uic.edu%2fowa%2f (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://parkingservices.uic.edu\n","Error occured:  HTTPSConnectionPool(host='parkingservices.uic.edu', port=443): Max retries exceeded with url: /Home (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://fm.red.uic.edu\n","Error occured:  HTTPSConnectionPool(host='fm.red.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://ovcas.red.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ovcas.red.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://logos.uic.edu/index.cgi\n","Error occured:  HTTPSConnectionPool(host='logos.uic.edu', port=443): Max retries exceeded with url: /index.cgi (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://red.uic.edu\n","Error occured:  HTTPSConnectionPool(host='red.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://fmweb.fm.uic.edu\n","Error occured:  HTTPConnectionPool(host='fmweb.fm.uic.edu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5aa45ea250>: Failed to establish a new connection: [Errno -2] Name or service not known'))  \n","\n","Unable to connect to  https://clery.uic.edu\n","Error occured:  HTTPSConnectionPool(host='clery.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://studyabroad.red.uic.edu/advising\n","Error occured:  HTTPSConnectionPool(host='studyabroad.red.uic.edu', port=443): Max retries exceeded with url: /advising (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://oia.uic.edu/about/oia-programs/international-agreements/active-uic-international-agreements\n","Error occured:  HTTPSConnectionPool(host='oia.uic.edu', port=443): Max retries exceeded with url: /about/oia-programs/international-agreements/active-uic-international-agreements (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa5829d90>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  https://oia.uic.edu/files/2014/06/IDF-AY20-21_APPLICATION-FORM-1-1.pdf\n","Error occured:  HTTPSConnectionPool(host='oia.uic.edu', port=443): Max retries exceeded with url: /files/2014/06/IDF-AY20-21_APPLICATION-FORM-1-1.pdf (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa09bb650>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  https://oia.uic.edu/files/2020/09/CCC-AWARDS-AY20-21APPLICATION-FORM.pdf\n","Error occured:  HTTPSConnectionPool(host='oia.uic.edu', port=443): Max retries exceeded with url: /files/2020/09/CCC-AWARDS-AY20-21APPLICATION-FORM.pdf (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa09ab850>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  https://oia.uic.edu/files/2020/09/DRUMRIGHT-AWARDS-AY20_21-APPLICATION-FORM.pdf\n","Error occured:  HTTPSConnectionPool(host='oia.uic.edu', port=443): Max retries exceeded with url: /files/2020/09/DRUMRIGHT-AWARDS-AY20_21-APPLICATION-FORM.pdf (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa09a0190>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  https://oia.uic.edu/files/2020/09/JNC-STUDENTS-AY20-21_APPLICATION-FORM.pdf\n","Error occured:  HTTPSConnectionPool(host='oia.uic.edu', port=443): Max retries exceeded with url: /files/2020/09/JNC-STUDENTS-AY20-21_APPLICATION-FORM.pdf (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa09a0ad0>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  https://listserv.uic.edu/cgi-bin/wa\n","Error occured:  HTTPSConnectionPool(host='listserv.uic.edu', port=443): Max retries exceeded with url: /cgi-bin/wa (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ask.publish.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ask.publish.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa4025d10>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  https://grad.red.uic.edu/programs/certificate-in-the-foundations-of-college-instruction\n","Error occured:  HTTPSConnectionPool(host='grad.red.uic.edu', port=443): Max retries exceeded with url: /programs/certificate-in-the-foundations-of-college-instruction (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ce.pharmacy.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ce.pharmacy.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.psych.uic.edu/education/continuing-education\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: /education/continuing-education (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://coeinmch.uic.edu/professional-development/continuing-education-programming\n","Error occured:  HTTPSConnectionPool(host='coeinmch.uic.edu', port=443): Max retries exceeded with url: /professional-development/continuing-education-programming (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.psych.uic.edu/education/continuing-education/social-work-continuing-education\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: /education/continuing-education/social-work-continuing-education (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://eyecare.uic.edu\n","Error occured:  HTTPSConnectionPool(host='eyecare.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://asme.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='asme.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://bmes.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='bmes.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ewh.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ewh.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://esw.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='esw.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ieee.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ieee.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://iise.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='iise.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ostem.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ostem.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://sae.uic.edu\n","Error occured:  HTTPSConnectionPool(host='sae.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://sase.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='sase.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://swe.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='swe.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://vese.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='vese.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://etakappanu.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='etakappanu.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://pts.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='pts.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://tbp.students.uic.edu\n","Error occured:  HTTPSConnectionPool(host='tbp.students.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://web.bioe.uic.edu/bioe-wp/bioe-undergrad/conc_area_form_fa16present.pdf\n","Error occured:  HTTPSConnectionPool(host='web.bioe.uic.edu', port=443): Max retries exceeded with url: /bioe-wp/bioe-undergrad/conc_area_form_fa16present.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://web.bioe.uic.edu/bioe-wp/bioe-undergrad/conc_area_form_fa14sp16.pdf\n","Error occured:  HTTPSConnectionPool(host='web.bioe.uic.edu', port=443): Max retries exceeded with url: /bioe-wp/bioe-undergrad/conc_area_form_fa14sp16.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://web.bioe.uic.edu/bioe-wp/bioe-undergrad/conc_area_form_fa12sp14.pdf\n","Error occured:  HTTPSConnectionPool(host='web.bioe.uic.edu', port=443): Max retries exceeded with url: /bioe-wp/bioe-undergrad/conc_area_form_fa12sp14.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://web.bioe.uic.edu/bioe-wp/bioe-undergrad/curr_flow_fa16present.htm\n","Error occured:  HTTPSConnectionPool(host='web.bioe.uic.edu', port=443): Max retries exceeded with url: /bioe-wp/bioe-undergrad/curr_flow_fa16present.htm (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://web.bioe.uic.edu/bioe-wp/bioe-undergrad/curr_flow_fa14sp16.htm\n","Error occured:  HTTPSConnectionPool(host='web.bioe.uic.edu', port=443): Max retries exceeded with url: /bioe-wp/bioe-undergrad/curr_flow_fa14sp16.htm (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://web.bioe.uic.edu/bioe-wp/bioe-undergrad/curr_flow_fa12sp14.htm\n","Error occured:  HTTPSConnectionPool(host='web.bioe.uic.edu', port=443): Max retries exceeded with url: /bioe-wp/bioe-undergrad/curr_flow_fa12sp14.htm (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://docs.financialaid.uic.edu/docs/PDF_upload_guide.pdf\n","Error occured:  HTTPSConnectionPool(host='docs.financialaid.uic.edu', port=443): Max retries exceeded with url: /docs/PDF_upload_guide.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://docs.financialaid.uic.edu/docs/QUICK INFORMATION_UNDERGRADUATES.pdf\n","Error occured:  HTTPSConnectionPool(host='docs.financialaid.uic.edu', port=443): Max retries exceeded with url: /docs/QUICK%20INFORMATION_UNDERGRADUATES.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://docs.financialaid.uic.edu/docs/SAP_Appeal_Evaluation_Form.pdf\n","Error occured:  HTTPSConnectionPool(host='docs.financialaid.uic.edu', port=443): Max retries exceeded with url: /docs/SAP_Appeal_Evaluation_Form.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://docs.financialaid.uic.edu/docs/2019FACalendar.pdf\n","Error occured:  HTTPSConnectionPool(host='docs.financialaid.uic.edu', port=443): Max retries exceeded with url: /docs/2019FACalendar.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://docs.financialaid.uic.edu/docs/2018FACalendar.pdf\n","Error occured:  HTTPSConnectionPool(host='docs.financialaid.uic.edu', port=443): Max retries exceeded with url: /docs/2018FACalendar.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ure.uic.edu/facfws.php\n","Error occured:  HTTPSConnectionPool(host='ure.uic.edu', port=443): Max retries exceeded with url: /facfws.php (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://pharmacy.uic.edu/about/rockford-campus\n","Error occured:  HTTPSConnectionPool(host='pharmacy.uic.edu', port=443): Max retries exceeded with url: /about/rockford-campus (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://anand.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='anand.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://shahbazian.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='shahbazian.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://csun.uic.edu\n","Error occured:  HTTPSConnectionPool(host='csun.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://gagel.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='gagel.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://mazumder.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='mazumder.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://nde.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='nde.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://jameslee.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='jameslee.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://aeon.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='aeon.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://mmnl.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='mmnl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://nrl.ece.uic.edu/people.html\n","Error occured:  HTTPSConnectionPool(host='nrl.ece.uic.edu', port=443): Max retries exceeded with url: /people.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://isnl.lab.uic.edu/isnl-dr.-tolou-shokuhfar.html\n","Error occured:  HTTPSConnectionPool(host='isnl.lab.uic.edu', port=443): Max retries exceeded with url: /isnl-dr.-tolou-shokuhfar.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://bestbioe.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='bestbioe.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://esmailbeigi.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='esmailbeigi.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://aes.uic.edu/programs/presidents-award-program.html\n","Error occured:  HTTPSConnectionPool(host='aes.uic.edu', port=443): Max retries exceeded with url: /programs/presidents-award-program.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://papautsky.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='papautsky.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://amml.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='amml.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://cbrl.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='cbrl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://themirbod.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='themirbod.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://yayuepan.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='yayuepan.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://nicest.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='nicest.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://acoustics.mie.uic.edu\n","Error occured:  HTTPSConnectionPool(host='acoustics.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://meml.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='meml.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://translog.uic.edu\n","Error occured:  HTTPSConnectionPool(host='translog.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://nicest.lab.uic.edu/profiles/smida-besma\n","Error occured:  HTTPSConnectionPool(host='nicest.lab.uic.edu', port=443): Max retries exceeded with url: /profiles/smida-besma (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://nicest.lab.uic.edu/profiles/khaledian-seiran\n","Error occured:  HTTPSConnectionPool(host='nicest.lab.uic.edu', port=443): Max retries exceeded with url: /profiles/khaledian-seiran (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://nicest.lab.uic.edu/profiles/soury-hamza\n","Error occured:  HTTPSConnectionPool(host='nicest.lab.uic.edu', port=443): Max retries exceeded with url: /profiles/soury-hamza (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://andrew.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='andrew.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://mnftl.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='mnftl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://nde.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='nde.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://csun.uic.edu/index.html\n","Error occured:  HTTPSConnectionPool(host='csun.uic.edu', port=443): Max retries exceeded with url: /index.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://serl.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='serl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://gagel.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='gagel.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://shi.uic.edu\n","Error occured:  HTTPSConnectionPool(host='shi.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://ptlynch.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ptlynch.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://mmnl.engr.uic.edu\n","Error occured:  HTTPSConnectionPool(host='mmnl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://anand.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='anand.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://red.uic.edu/wp-signup.php\n","Error occured:  HTTPSConnectionPool(host='red.uic.edu', port=443): Max retries exceeded with url: /wp-signup.php (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://nanoeng.uic.edu\n","Error occured:  HTTPSConnectionPool(host='shahbazian.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://geotech.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='gagel.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/app-that-tracks-bipolar-manic-depressive-episodes-wins-award\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /app-that-tracks-bipolar-manic-depressive-episodes-wins-award (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://yayuepan.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='yayuepan.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://rc.uic.edu\n","Error occured:  HTTPSConnectionPool(host='acer.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/saber-builds-campus-computing-backbone\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /saber-builds-campus-computing-backbone (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/uic-bioengineer-receives-fulbright-to-study-in-finland\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /uic-bioengineer-receives-fulbright-to-study-in-finland (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/bioengineering-student-awarded-gates-cambridge-scholarship\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /bioengineering-student-awarded-gates-cambridge-scholarship (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/bubble-recoil-could-be-used-to-cool-microchips-even-in-space\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /bubble-recoil-could-be-used-to-cool-microchips-even-in-space (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://lore.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='lore.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://sae.uic.edu\n","Error occured:  HTTPSConnectionPool(host='sae.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://cbrl.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='cbrl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/5m-federal-grant-to-help-uic-graduate-more-latino-scientists-engineers\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /5m-federal-grant-to-help-uic-graduate-more-latino-scientists-engineers (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/can-cellphone-use-predict-manic-episodes-in-bipolar-disorder\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /can-cellphone-use-predict-manic-episodes-in-bipolar-disorder (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/new-nanomaterial-offers-promise-in-bendable-wearable-electronic-devices\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /new-nanomaterial-offers-promise-in-bendable-wearable-electronic-devices (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://mnftl.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='mnftl.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://kenbrez.lab.uic.edu\n","Error occured:  HTTPSConnectionPool(host='hpst.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://nanomaterialsenergysystems.engr.uic.edu\n","Error occured:  HTTPSConnectionPool(host='nanomaterialsenergysystems.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/graphene-sandwich-improves-imaging-of-biomolecules\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /graphene-sandwich-improves-imaging-of-biomolecules (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/graphene-nano-sensor-depends-on-defects-for-sensitivity\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /graphene-nano-sensor-depends-on-defects-for-sensitivity (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/graphene-heat-transfer-riddle-unravelled\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /graphene-heat-transfer-riddle-unravelled (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/supersonic-spray-delivers-high-quality-graphene-layer\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /supersonic-spray-delivers-high-quality-graphene-layer (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://cs.red.uic.edu/graduate/ms-program\n","Error occured:  HTTPSConnectionPool(host='cs.red.uic.edu', port=443): Max retries exceeded with url: /graduate/ms-program (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://cs.red.uic.edu/graduate/phd-program\n","Error occured:  HTTPSConnectionPool(host='cs.red.uic.edu', port=443): Max retries exceeded with url: /graduate/phd-program (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://cs.red.uic.edu/research/research-areas\n","Error occured:  HTTPSConnectionPool(host='cs.red.uic.edu', port=443): Max retries exceeded with url: /research/research-areas (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://cs.red.uic.edu/graduate/post-grad-outcomes\n","Error occured:  HTTPSConnectionPool(host='cs.red.uic.edu', port=443): Max retries exceeded with url: /graduate/post-grad-outcomes (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://hci.cs.uic.edu/\n","Error occured:  HTTPSConnectionPool(host='hci.cs.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://nlp.lab.uic.edu/\n","Error occured:  HTTPSConnectionPool(host='nlp.lab.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://flamestutoring.uic.edu\n","Error occured:  HTTPSConnectionPool(host='flamestutoring.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.psych.uic.edu/clinical/adult-services\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: /clinical/adult-services (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ossp.uic.edu/scholarships\n","Error occured:  HTTPSConnectionPool(host='ossp.uic.edu', port=443): Max retries exceeded with url: /scholarships (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/disability-ethics-certificate-program\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/disability-ethics-certificate-program (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://http://ahs.uic.edu\n","Error occured:  HTTPConnectionPool(host='http', port=80): Max retries exceeded with url: //ahs.uic.edu (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5aa4949710>: Failed to establish a new connection: [Errno -2] Name or service not known'))  \n","\n","Unable to connect to  https://idhd-ahslabs.red.uic.edu/training/ceus\n","Error occured:  HTTPSConnectionPool(host='idhd-ahslabs.red.uic.edu', port=443): Max retries exceeded with url: /training/ceus (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ccspd.org.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ccspd.org.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa216c590>: Failed to establish a new connection: [Errno -2] Name or service not known'))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/dhd-559-ergonomics-safety-for-workers-with-disabilities\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/dhd-559-ergonomics-safety-for-workers-with-disabilities (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/dhd-562-mobile-technology-computers-built-in-accessibility-features\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/dhd-562-mobile-technology-computers-built-in-accessibility-features (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/dhd-567-aac-for-individuals-with-autism-spectrum-disorder\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/dhd-567-aac-for-individuals-with-autism-spectrum-disorder (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/dhd-569-environmental-modification\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/dhd-569-environmental-modification (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/dhd-594a-technology-to-support-universal-design-for-learning\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/dhd-594a-technology-to-support-universal-design-for-learning (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/3d-printing-in-assistive-technology\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/3d-printing-in-assistive-technology (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/introduction-assistive-technology-principles-practice\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/introduction-assistive-technology-principles-practice (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/dhd-548-assistive-technology-tools-in-the-pk-12-educational-setting\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/dhd-548-assistive-technology-tools-in-the-pk-12-educational-setting (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/assistive-technology-for-individuals-who-are-blind-or-visually-impaired\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/assistive-technology-for-individuals-who-are-blind-or-visually-impaired (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/dhd-554-augmentative-communication-assessment\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/dhd-554-augmentative-communication-assessment (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/consideration-assessment-documentation-assistive-technology-prek-12-educational-setting\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/consideration-assessment-documentation-assistive-technology-prek-12-educational-setting (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/introduction-to-microcontrollers-in-assistive-technology\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/introduction-to-microcontrollers-in-assistive-technology (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/seating-positioning-wheelchair-mobility\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/seating-positioning-wheelchair-mobility (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/manual-wheelchair-technology\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/manual-wheelchair-technology (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/supporting-augmentative-and-alternative-communication-in-educational-settings\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/supporting-augmentative-and-alternative-communication-in-educational-settings (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/applied-health-sciences/computers-communication-controls-rehabilitation-technology\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /applied-health-sciences/computers-communication-controls-rehabilitation-technology (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.psych.uic.edu/research/center-on-mental-health-services-research-and-policy/about-us/about-the-centerpublications.htm\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: /research/center-on-mental-health-services-research-and-policy/about-us/about-the-centerpublications.htm (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://las110.class.uic.edu\n","Error occured:  HTTPSConnectionPool(host='las110.class.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ertc.chem.uic.edu\n","Error occured:  HTTPSConnectionPool(host='ertc.chem.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://www.uic.edu/uic/studentlife/campus/emergency-information.shtml\n","Error occured:  HTTPSConnectionPool(host='emergency.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://hsi.uic.edu\n","Error occured:  HTTPSConnectionPool(host='hsi.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://cas.uic.edu/upass_pickup/index2.php\n","Error occured:  HTTPSConnectionPool(host='cas.uic.edu', port=443): Max retries exceeded with url: /upass_pickup/index2.php (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://go.uic.edu/spring_upass\n","Error occured:  Exceeded 30 redirects.  \n","\n","Unable to connect to  https://convocation.uic.edu\n","Error occured:  HTTPSConnectionPool(host='convocation.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://wow.uic.edu\n","Error occured:  HTTPSConnectionPool(host='studentaffairs.uic.edu', port=443): Max retries exceeded with url: /wow (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://cas.uic.edu/wordpresspreview/scw/scw.html\n","Error occured:  HTTPSConnectionPool(host='cas.uic.edu', port=443): Max retries exceeded with url: /wordpresspreview/scw/scw.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://cas.uic.edu/wordpresspreview/sce/sce.html\n","Error occured:  HTTPSConnectionPool(host='cas.uic.edu', port=443): Max retries exceeded with url: /wordpresspreview/sce/sce.html (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://studentcenters.red.uic.edu/wp-content/uploads/sites/159/2017/11/Green-Office-Guide.pdf\n","Error occured:  HTTPSConnectionPool(host='studentcenters.red.uic.edu', port=443): Max retries exceeded with url: /wp-content/uploads/sites/159/2017/11/Green-Office-Guide.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://vcsa.uic.edu\n","Error occured:  HTTPSConnectionPool(host='vcsa.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://commuter.uic.edu/housing\n","Error occured:  HTTPSConnectionPool(host='commuter.uic.edu', port=443): Max retries exceeded with url: /housing (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://satestb.red.uic.edu/wp-content/uploads/sites/10/2017/11/AOD_UPDATED_9517.pdf\n","Error occured:  HTTPSConnectionPool(host='satestb.red.uic.edu', port=443): Max retries exceeded with url: /wp-content/uploads/sites/10/2017/11/AOD_UPDATED_9517.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://satestb.red.uic.edu/wp-content/uploads/sites/10/2017/11/Campus-Violence-Response-and-Prevention-PlanUniversity-of-Illinoi.pdf\n","Error occured:  HTTPSConnectionPool(host='satestb.red.uic.edu', port=443): Max retries exceeded with url: /wp-content/uploads/sites/10/2017/11/Campus-Violence-Response-and-Prevention-PlanUniversity-of-Illinoi.pdf (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://www.uic.edu/pharmacy\n","Error occured:  HTTPSConnectionPool(host='pharmacy.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/staff/sherri-mcginnis-gonzalez/120213_mcginnis_gonzalez_030\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /staff/sherri-mcginnis-gonzalez/120213_mcginnis_gonzalez_030 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/staff/jeffron-boynes/120213_boynes_015\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /staff/jeffron-boynes/120213_boynes_015 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/staff/brian-flood/120213_flood_027\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /staff/brian-flood/120213_flood_027 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/staff/christy-levy/120213_levy_026\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /staff/christy-levy/120213_levy_026 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/courses\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /courses (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/programs\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /programs (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/cnm\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /cnm (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/grant-writing-workshop\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /grant-writing-workshop (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/social-enterprise-certificate\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /social-enterprise-certificate (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/partner-with-us\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /partner-with-us (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/request-information\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /request-information (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/about\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /about (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/cnm/courses/donor-relationships\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /cnm/courses/donor-relationships (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/cnm/courses/nonprofits-today\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /cnm/courses/nonprofits-today (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/cnm/courses\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /cnm/courses (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/grant-writing-workshop/noah-temaner-jenkins\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /grant-writing-workshop/noah-temaner-jenkins (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://ec.uic.edu/grant-writing-workshop/valerie-leonard\n","Error occured:  HTTPSConnectionPool(host='ec.uic.edu', port=443): Max retries exceeded with url: /grant-writing-workshop/valerie-leonard (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/files/2014/11/IMG_8600.jpg\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /files/2014/11/IMG_8600.jpg (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://news.uic.edu/experts/zizi-papacharissi/zizi-papacharissi-2\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /experts/zizi-papacharissi/zizi-papacharissi-2 (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://https://comm.uic.edu/profiles/zizi-papacharissi\n","Error occured:  HTTPConnectionPool(host='https', port=80): Max retries exceeded with url: //comm.uic.edu/profiles/zizi-papacharissi (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5a9e2e1d10>: Failed to establish a new connection: [Errno -2] Name or service not known'))  \n","\n","Unable to connect to  http://news.uic.edu/files/2016/02/Weine.Stevan27.jpg\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /files/2016/02/Weine.Stevan27.jpg (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://www.psych.uic.edu/154-about-us/directory/faculty/266-steve-weine\n","Error occured:  HTTPSConnectionPool(host='www.psych.uic.edu', port=443): Max retries exceeded with url: /154-about-us/directory/faculty/266-steve-weine (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://findingaids.library.uic.edu/sc/msscic94.xml\n","Error occured:  HTTPSConnectionPool(host='findingaids.library.uic.edu', port=443): Max retries exceeded with url: /sc/msscic94.xml (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.hr.uic.edu/classification_and_compensation/about_civil_service\n","Error occured:  HTTPSConnectionPool(host='www.hr.uic.edu', port=443): Max retries exceeded with url: /classification_and_compensation/about_civil_service (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://www.hr.uic.edu/managing_my_performance/annual_performance_review\n","Error occured:  HTTPSConnectionPool(host='www.hr.uic.edu', port=443): Max retries exceeded with url: /managing_my_performance/annual_performance_review (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://groups.accc.uic.edu\n","Error occured:  HTTPSConnectionPool(host='groups.accc.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://campusprograms.uic.edu\n","Error occured:  HTTPSConnectionPool(host='campusprograms.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: certificate has expired (_ssl.c:1091)')))  \n","\n","Unable to connect to  http://news.uic.edu/files/2015/10/DG15_04_50_005.jpg\n","Error occured:  HTTPSConnectionPool(host='news.uic.edu', port=443): Max retries exceeded with url: /files/2015/10/DG15_04_50_005.jpg (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n","Unable to connect to  https://courseval.den.uic.edu/etw/ets/et.asp\n","Error occured:  HTTPSConnectionPool(host='courseval.den.uic.edu', port=443): Max retries exceeded with url: /etw/ets/et.asp (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa4017cd0>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  http://portal.den.uic.edu\n","Error occured:  HTTPConnectionPool(host='portal.den.uic.edu', port=80): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x7f5aa298f490>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  https://portal.den.uic.edu\n","Error occured:  HTTPSConnectionPool(host='portal.den.uic.edu', port=443): Max retries exceeded with url: / (Caused by NewConnectionError('<urllib3.connection.VerifiedHTTPSConnection object at 0x7f5aa29bfb90>: Failed to establish a new connection: [Errno 110] Connection timed out'))  \n","\n","Unable to connect to  http://bsn.ec.uic.edu\n","Error occured:  HTTPSConnectionPool(host='bsn.ec.uic.edu', port=443): Max retries exceeded with url: / (Caused by SSLError(SSLCertVerificationError(1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1091)')))  \n","\n"],"name":"stdout"}]},{"cell_type":"code","metadata":{"id":"CKTAW1FDI6zP","colab":{"base_uri":"https://localhost:8080/"},"executionInfo":{"status":"ok","timestamp":1619891484206,"user_tz":300,"elapsed":293,"user":{"displayName":"Swathi Bodavula","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjEnLhW6QQwYx5lzdBRurwQeXMmO-cdAZHAmMnI=s64","userId":"10131213081504042480"}},"outputId":"21c077d6-9d5d-4925-fa82-1146472d2240"},"source":["pickle_folder = \"./Pickle_Folder/\"\n","os.makedirs(pickle_folder, exist_ok=True)\n","\n","# Pickling the dict of crawled pages\n","with open(pickle_folder + '6000_url_pages.pickle', 'wb') as f:\n","    pickle.dump(crawled_pages,f)\n","\n","\n","with open(pickle_folder + '6000_url_pages.pickle', 'rb') as f:\n","    url_pages = pickle.load(f)\n","\n","crawled_pages == url_pages\n"],"execution_count":null,"outputs":[{"output_type":"execute_result","data":{"text/plain":["True"]},"metadata":{"tags":[]},"execution_count":18}]},{"cell_type":"code","metadata":{"colab":{"base_uri":"https://localhost:8080/"},"id":"j-iXZHxJ_uEl","executionInfo":{"status":"ok","timestamp":1619894970254,"user_tz":300,"elapsed":533,"user":{"displayName":"Swathi Bodavula","photoUrl":"https://lh3.googleusercontent.com/a-/AOh14GjEnLhW6QQwYx5lzdBRurwQeXMmO-cdAZHAmMnI=s64","userId":"10131213081504042480"}},"outputId":"42744c47-9613-4448-8217-646e470a9894"},"source":["len(crawled_pages)\n","    "],"execution_count":null,"outputs":[{"output_type":"execute_result","data":{"text/plain":["6001"]},"metadata":{"tags":[]},"execution_count":19}]},{"cell_type":"code","metadata":{"id":"g4Yn72MICFk5"},"source":[""],"execution_count":null,"outputs":[]}]}